airbyte-integrations/connectors/destination-google-sheets/destination_google_sheets/buffer.py

```diff
@@ -0,0 +1,124 @@
+
+    def flush_buffer(self, stream_name: str):
+        """
+        Cleans up the `records_buffer` values, belonging to input stream.
+        """
+        [stream[stream_name].clear() for stream in self.records_buffer if stream_name in stream]
+
+    def normalize_record(self, stream_name: str, record: Mapping) -> Mapping[str, Any]:
+        """
+        Updates the record keys up to the input configured_stream catalog keys.
+
+        Handles two scenarios:
+        1) when record has less keys than catalog declares (undersetting)
+        2) when record has more keys than catalog declares (oversetting)
+
+        Returns: alphabetically sorted, catalog-normalized Mapping[str, Any].
+
+        EXAMPLE:
+        - UnderSetting:
+            * Catalog:
+                - has 3 entities:
+                    [ 'id', 'key1', 'key2' ]
+                              ^
+            * Input record:
+                - missing 1 entity, compare to catalog
+                    { 'id': 123,    'key2': 'value' }
+                                  ^
+            * Result:
+                - 'key1' has been added to the record, because it was declared in catalog, to keep the data structure.
+                    {'id': 123, 'key1': '', {'key2': 'value'} }
+                                  ^
+        - OverSetting:
+            * Catalog:
+                - has 3 entities:
+                    [ 'id', 'key1', 'key2',   ]
+                                            ^
+            * Input record:
+                - doesn't have entity 'key1'
+                - has 1 more enitity, compare to catalog 'key3'
+                    { 'id': 123,     ,'key2': 'value', 'key3': 'value' }
+                                  ^                      ^
+            * Result:
+                - 'key1' was added, because it expected be the part of the record, to keep the data structure
+                - 'key3' was dropped, because it was not declared in catalog, to keep the data structure
+                    { 'id': 123, 'key1': '', 'key2': 'value',   }
+                                   ^                          ^
+
+        """
+
+        for stream in self.stream_info:
+            if stream_name in stream:
+                # undersetting scenario
+                [record.update({key: self.default_missing}) for key in stream[stream_name] if key not in record.keys()]
+                # oversetting scenario
+                [record.pop(key) for key in record.copy().keys() if key not in stream[stream_name]]
+
+        return dict(sorted(record.items(), key=lambda x: x[0]))
+
+    def get_record_values(self, record: Mapping) -> List[str]:
+        """
+        Returns the input record values.
+        """
+        return self.values_to_str(list(record.values()))
+
+    def values_to_str(self, values: List) -> List[Any]:
+        """
+        Force input record values to be a type of string.
+        """
+        return [str(values[i]) for i in range(len(values))]
```

Comment:
list(map(str, values))

or 

return [str(value) for value in values]

Reply:
Done.